packages/voting-subgraph/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "aragon-voting-subgraph",
+  "version": "0.1.0",
+  "scripts": {
+    "codegen": "rm -rf generated; graph codegen",
+    "manifest": "scripts/build-manifest.sh mainnet",
+    "manifest-staging": "env STAGING=true scripts/build-manifest.sh mainnet",
+    "manifest-rinkeby": "scripts/build-manifest.sh rinkeby",
+    "manifest-rinkeby-staging": "env STAGING=true scripts/build-manifest.sh rinkeby",
+    "deploy": "scripts/deploy.sh ajsantander voting mainnet",
+    "deploy-staging": "env STAGING=true scripts/deploy.sh ajsantander voting mainnet"
```

Comment:
Missed a comma here!

Reply:
Dawm bro... you got a linter in your eyes.